docker/dotcms/ROOT/srv/20-copy-overriden-files.sh

```diff
@@ -2,5 +2,9 @@
 
 set -e
 
+[[ -f /srv/TOMCAT_VERSION ]] && TOMCAT_VERSION=$( cat /srv/TOMCAT_VERSION )
+
 cp /srv/OVERRIDE/tomcat/conf/*      $TOMCAT_HOME/conf/
+mv $TOMCAT_HOME/conf/server-${TOMCAT_VERSION}.xml $TOMCAT_HOME/conf/server.xml && \
```

Comment:
I think the fix is to change the filename from `server-${TOMCAT_VERSION}.xml` to `server.xml`.  With the Dockerfile in the core repo, we do not need to support multiple versions of tomcat in any given branch and can remove all that  conditional logic.